eclair-core/src/test/scala/fr/acinq/eclair/TestkitBaseClass.scala

```diff
@@ -30,10 +30,6 @@ import scala.concurrent.Await
   */
 abstract class TestkitBaseClass extends TestKit(ActorSystem("test")) with fixture.FunSuiteLike with BeforeAndAfterEach with BeforeAndAfterAll {
```

Comment:
You can remove the `BeforeAndAfterAll` trait

Reply:
I can't, we override `afterEach` and `afterAll`.

Reply:
Oh yeah I confused that with one that I added in my amp branch (and that I can remove once this is on master)